libfreerdp/core/settings.c

```diff
@@ -191,15 +193,19 @@ void settings_load_hkey_local_machine(rdpSettings* settings)
 		settings_client_load_hkey_local_machine(settings);
 }
 
-void settings_get_computer_name(rdpSettings* settings)
+BOOL settings_get_computer_name(rdpSettings* settings)
 {
-	DWORD nSize = 0;
+	DWORD nSize = MAX_COMPUTERNAME_LENGTH + 1;
+	TCHAR computerName[MAX_COMPUTERNAME_LENGTH + 1];
```

Comment:
use CHAR instead (since GetComputerNameExA and other ascii function versions are used below)